indra/llrender/llglslshader.cpp

```diff
@@ -774,25 +801,21 @@ GLint LLGLSLShader::mapUniformTextureChannel(GLint location, GLenum type, GLint
         if (size == 1)
         {
             glUniform1i(location, mActiveTextureChannels);
-            LL_DEBUGS("ShaderUniform") << "Assigned to texture channel " << mActiveTextureChannels << LL_ENDL;
             mActiveTextureChannels++;
         }
         else
         {
             //is array of textures, make sequential after this texture
-            GLint channel[32]; // <=== only support up to 32 texture channels
-            llassert(size <= 32);
-            size = llmin(size, 32);
+            GLint channel[16]; // <=== only support up to 16 texture channels
```

Comment:
probably will want a big global `static const U32 MAX_SUPPORTED_TEXTURE_CHANNELS` somewhere for this